docs/firewall.md

```diff
@@ -0,0 +1,45 @@
+
+Our firewall has protected customers against active attacks, including HTTP request floods up to 1 million requests per second, sustained over hours.
+
+
+## Protection against exploits
+
+The Altis WAF includes protection against web exploits, including Cross-Site Scripting (XSS), SQL injections (SQLi), and other known inputs. It also includes protection against exfiltration of private files accidentally published in your codebase, including .sql and .sh files.
+
+The Altis team manages these protections automatically across the platform on your behalf.
+
+
+## Protection against request floods & denial of service attacks
+
+The Altis WAF provides protection against denial of service (DoS) and distributed denial of service (DDoS) attacks.
+
+Per-IP rate limits are applied across all environments as standard, both across the entire environment as well as per-container, to ensure individual bad actors are limited from being able to adversely affect your site. Rate limits are monitored and adjusted across the platform depending on industry-wide threat analysis; contact Altis support for further information about the currently-applied rate limits.
+
+Altis also includes advanced DDoS protection as standard for our customers. We actively monitor all environments for attacks, with automated interventions and mitigations. Our engineers are on-call 24/7 to respond if necessary, and we work with the AWS Shield Response Team (SRT) where needed to mitigate internet-scale attacks.
+
+For customers who experience persistent DDoS attacks, Altis will work with your team to put additional mitigations in place. For example, stricter rate limits on backend requests (such as `/wp-admin`) may be put in place.
+
+
+## What if my IP address gets blocked?
+
+Our system operates automatically to detect and mitigate threats. In some cases, this can lead to legitimate users being blocked.
+
+If you believe your IP address has been blocked accidentally, contact Altis support, who can investigate why an IP address may be blocked.
+
+For customers with additional DDoS mitigations in place, legitimate backend users (such as editors and site admins) may be blocked at a higher rate than normal. As part of organizing additional mitigations, Altis engineers can relax these mitigations for your users. (Note that regular firewall rules may still apply.)
+
+Advanced bypassing of our firewall, such as allowing specific IP addresses through ("whitelisting"), requires firewall customisation.
+
+
+## Firewall customizations
+
+Our firewall is maintained as standard across our platform, and for most customers, you'll never need to worry about your site. Our platform also provides a variety of options for limiting access further, such as [custom nginx configuration](./nginx-configuration.md).
```

Comment:
can link to examples specifically in https://github.com/humanmade/altis-cloud/pull/640